docroot/modules/humsci/hs_admin/hs_admin.libraries.yml

```
@@ -5,3 +5,8 @@ hs_admin:
       assets/css/hs_admin.css: {}
   js:
     assets/js/hs_admin.js: {}
+
+default_shortcuts:
+  version: VERSION
```

```suggestion
  version: VERSION
```

I would drop the `version: VERSION` from library definitions unless you intend to use a version and update it with every change. We discovered a [recent change](https://www.drupal.org/node/3301716) in Drupal core 10.1+ doesn't re-aggregate library files unless the library version is changed. Using `VERSION` here as the version reflects the Drupal core version -- if Drupal Core does not get upgraded in a release, it wouldn’t re-aggregate the CSS/JS in the library.

---

@cienvaras

---

@joegl Thanks, fixed.